### ISSUE TITLE
MongoChangeStreamWorker uses thread.stop() which is deprecated

### ISSUE THREAD

**Issue by piotr-bugara-gravity9** [gravity9-tech/mongocdc#9]
Method `stop()` in MongoChangeStreamWorker uses `thread.stop()` which is deprecated and marked for removal.
Current implementation of `thread.stop()` method is
```@Deprecated(since="1.2", forRemoval=true)
    public final void stop() {
        throw new UnsupportedOperationException();
    }

**Comment by PiotrGravity** [comment]
I'll try to handle it

**Comment by PiotrGravity** [comment]
Resolved by #38
